package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "openexp",
-  "version": "0.1.2",
+  "version": "0.1.3",
   "description": "An open-source mean stack app for running experiments with openbci",
```

Comment:
do we want to keep the description with "...with openbci"? or should we wait to change it later when we support more.